public/style.css

```diff
@@ -63,4 +63,101 @@ table {
     margin:5px;
     background:darkgray;
     color:black;
-}
\ No newline at end of file
+}
```

Comment:
References: 
1) https://stackoverflow.com/questions/39846282/how-to-add-the-text-on-and-off-to-toggle-button
2) https://www.w3schools.com/howto/howto_css_switch.asp

Should I include this at the top of the file?

Reply:
I think so. The way we did it in nparser.py was:
__source__ = '''
https://github.com/pyparsing/pyparsing/blob/master/examples/fourFn.py
'''

We should probably do the same here.

Reply:
I couldn't really find the example in the code anymore since it seems to have changed but I updated to include this at the top of the css file. Hope its ok.